### PR TITLE
added a static data for test

### PR DESCRIPTION
**Pull request by vadim11041996**
added a few static data that Elena will need to test

### CODE REVIEW

File: src/seeders/20180704134412-demo-teacher.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ module.exports = {
 
 function generate50Teachers() {
   const teachers = [];
+  teachers.push({
+    first_name: 'teacher',
+    last_name: 'teacher',
+    email: 'teacher@email.com',
+    password: 'teacher',
```
Comment (alxolr) [inline src/seeders/20180704134412-demo-teacher.js]: Passwords should be hashed, otherwise it will not work

File: api/swagger/swagger.yaml [hunk 2]
```diff
@@ -133,7 +133,7 @@ paths:
           description: "success"
           schema:
             type: "string"
-        400:
+        404:
```
Comment (alxolr) [inline api/swagger/swagger.yaml]: We agreed that if you have a refactoring to do you create a separate pull request. One pull request should have one reason or ticket.

Comment (vadim11041996) [reply api/swagger/swagger.yaml]: yes, i've changed it yesterday and forgot, my bad

File: src/seeders/20180704134412-demo-teacher.js (hunk 1 above)
Comment (vadim11041996) [reply src/seeders/20180704134412-demo-teacher.js]: 1 min

Comment (alxolr) [reply src/seeders/20180704134412-demo-teacher.js]: Good Job @thirteenmd , but still can't accept the pull request if it is not on versionned
